<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMitra</class>
 <widget class="QWidget" name="FormMitra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>171</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="kd_mitra"/>
    </item>
    <item>
     <widget class="QLineEdit" name="nama_mitra"/>
    </item>
    <item>
     <widget class="QLineEdit" name="alamat_mitra"/>
    </item>
    <item>
     <widget class="QLineEdit" name="telepon_mitra"/>
    </item>
    <item>
     <widget class="QLineEdit" name="email_mitra"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>HALAMAN ADMIN</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>383</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnTambah">
      <property name="text">
       <string>Tambah</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEdit">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBersih">
      <property name="text">
       <string>BERSIH</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnHapus">
      <property name="text">
       <string>Hapus</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>160</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kode Mitra :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nama Mitra :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Alamat Mitra :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>No telpon :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Email :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="editcari">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>169</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>158</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pencarian :</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableMitra">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>430</y>
     <width>381</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>kd_mitra</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>nama_mitra</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>alamat_mitra</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>telepon_mitra</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>email_mitra</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
